### ISSUE TITLE
32-bit ITP files

### ISSUE THREAD

**Issue by shinkiseki** [shinkiseki/MorePortraitsInZero#3]
Currently, new portraits have been created using the [png32toitx Python script](https://github.com/AGraber/EDDecompiler/blob/master/Decompiler/png32toitx.py). However, this appears to generate an 8-bit ITP file, rather than a 32-bit ITP file. If/when a tool for compressing images into 32-bit ITP images is developed, the portrait files will be recreated.

**Comment by Kyuuhachi** [comment]
I've made [a script](https://gist.github.com/Kyuuhachi/5af2b9c2036ae8e9b474e43b2854eef3) that could help with this. I couldn't find any python library for bc7 compression, so it can only convert already bc7-encoded DDS files, but it might be of use regardless.

**Comment by shinkiseki** [comment]
Just from a cursory Google search, it looks like there are plenty of public tools for converting PNG to DDS. Those tools didn't specify that they were for bc7-encoded DDS files, though, so I'm skeptical that they'd be the right move. That being said, my experience with textures is basically zero (hah), so I'd defer to someone that knows more than me.

**Comment by Kyuuhachi** [comment]
Yeah, I don't know what tool is good for that part. I used paint.net for Inevitable, but I only had a single texture to deal with, so I didn't have any reason to look into a batch converter.

Some googling finds [bc7enc](https://github.com/richgel999/bc7enc_rdo) (no prebuilt releases, commandline only) and [Compressionator](https://github.com/GPUOpen-Tools/Compressonator/releases) (which has neither of those shortcomings).

It's not at all unlikely that NISA-itp does support other encodings too, but I have no idea what bytes would specify this.

**Comment by shinkiseki** [comment]
Will update portraits to 32-bit ITP thanks to Kyuuhachi's DDS-to-ITP converter in a future update.

Link to forked code with cmdline functionality.
https://gist.github.com/shinkiseki/059eb28ee38e581fd8973c8d647f345d
